api/sample_data.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO UnitListerAccount
```

Comment:
if you change types in the  create_tables.sql file, don't forget to change these lines e.g. changing from string type to enum

Reply:
It still shows up as a string when you insert, the ENUM type checks the string is in the ENUM.